tickets/models.py

```
@@ -1,17 +1,48 @@
+    timestamp: datetime
+    selection_data: Optional[PickNSelectionData] = Field(None, description="User's picks for 'Pick N' games")
+
+    class Config:
+        populate_by_name = True
+        json_encoders = {
+            datetime: lambda dt: dt.isoformat(),
+        }
+        model_config = {
+            "from_attributes": True,
+            "populate_by_name": True,
+             "json_encoders": {
```

Flake8 (7.2.0)

[error] continuation line unaligned for hanging indent

(E131)